app/services/weather_service.rb

```diff
@@ -0,0 +1,16 @@
+class WeatherService
+  
+  def initialize
+    @conn ||= Faraday.new(url: "http://api.wunderground.com/api/#{ENV['WEATHER_KEY']}") do |faraday|
+      faraday.adapter Faraday.default_adapter
+    end
+  end
+
+  def weather_info(lat,lon)
+    parse(@conn.get("conditions/q/#{lat},#{lon}.json"))
+  end
+
+  def parse(response)
+    JSON.parse(response.body, symbolize_names: true)
+  end
+end
```

Comment:
@Mcents Service looks great, I like how `parse(response)` could be reused easily if we make additional requests from wunderground 👍